File: Beecrowd/1161 Soma de Fatoriais/main.cpp
#include <iostream>

using namespace std;

long long fatorial(int n);

int main()
{
    long long soma;
    int m, n;

    while (cin >> m >> n)
    {

        soma = fatorial(m) + fatorial(n);

        cout << soma << endl;
    }

    return 0;
}

long long fatorial(int n)
{
    long long f = 1;

    for (int i = n; i >= 1; i--)
        f *= i;

    return f;
}

File: Beecrowd/1161 Soma de Fatoriais/main.py
def fatorial(n):
    f = 1

    for i in range(n, 1, -1):
        f *= i

    return f


while True:
    try:
        m, n = (int(_) for _ in input().split())

        soma = fatorial(m) + fatorial(n)

        print(soma)

    except EOFError:
        break
